src/show.jl

```diff
@@ -132,26 +133,51 @@ const tensorboard = Ref{Tensorboard}()
 
 function Base.close(board::Tensorboard)
     kill(board.proc, 2)
-    #rm(board.logdir, recursive=true, force=true)
 end
 
-function get_tensorboard(logdir=nothing)
-    if true#!isdefined(tensorboard, :x) || (logdir !== nothing && tensorboard[].logdir ≠ logdir)
-        if isdefined(tensorboard, :x)
-            port = tensorboard[].port + 1
-        else
-            port = 6006
-        end
-        if logdir === nothing
-            logdir = mktempdir()
-        end
-        _, proc = open(`tensorboard --logdir=$logdir --port=$port`)
-        tensorboard[] = Tensorboard(proc, logdir, port)
-        atexit() do
-            close(tensorboard[])
+"""
+    find_tensorboard()
+
+Return the path to the tensorboard executable.
+
+Checks everywhere in the system path, as well in the binary python directory
+associated with the Conda.jl installation, should it exist.
+"""
+function find_tensorboard()
+    path = nothing
+    dirs = split(ENV["PATH"], ":")
+    if PyCall.conda
+        @eval import Conda
+        push!(dirs, Conda.BINDIR)
```

Comment:
Rather than using `@eval` to bring load Conda into `main`.
Maybe just use it via where it is already imported in PyCall.

i.e. `push!(dirs, PyCall.Conda.BINDIR)`

This feels less dirty to me than `@eval`